pkg/trace/agent/normalizer.go

```diff
@@ -194,6 +195,16 @@ func setChunkAttributesFromRoot(chunk *pb.TraceChunk, root *pb.Span) {
 		// Older tracers set origin in the root span.
 		chunk.Origin = root.Meta[tagOrigin]
 	}
+
+	if _, ok := chunk.Tags[tagDecisionMaker]; !ok {
+		for _, span := range chunk.Spans {
+			if dm, ok := span.Meta[tagDecisionMaker]; ok {
+				chunk.Tags[tagDecisionMaker] = dm
+				//todo: delete the dm on the span we found?
```

Comment:
I don't think there's a need to keep it. However, if more than one span has `tagDecisionMaker`, we only find, use, and delete the first. Could cause confusion, especially if the one we actually used is deleted.

Maybe we want to delete *all* `tagDecisionMaker`?

Reply:
yeah that makes sense to me (and ideally we can convince tracers to migrate to putting them on the chunk itself